core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2405,7 +2405,8 @@ class SparkContext(config: SparkConf) extends Logging {
     ResourceProfile.clearDefaultProfile()
     // Unset YARN mode system env variable, to allow switching between cluster types.
     SparkContext.clearActiveContext()
-    logInfo("Successfully stopped SparkContext")
+    logInfo(log"Successfully stopped SparkContext (Uptime: " +
```

Comment:
A minor objection here that `startTime` and this calculation are derived from `System.currentTimeMllis()`, which is subject to clock skew and system time resets. However, I haven't investigated what it would take to migrate this code path to the more reliable `System#nanoTime()`.

Regardless, a nice improvement, and +1 (non-binding)!